forge-gui/res/cardsfolder/upcoming/phyresis_roach.txt

```diff
@@ -0,0 +1,12 @@
+Name:Phyresis Roach
+ManaCost:G
+Types:Creature Phyrexian Insect
+PT:1/1
+K:Toxic:1
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player | CombatDamage$ True | Execute$ TrigEffect | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, Insects you control and Insect cards in your graveyard, hand, and library perpetually gain toxic 1.
+SVar:TrigEffect:DB$ Effect | Name$ Phyresis Roach's Perpetual Effect | RememberObjects$ Insect.YouCtrl,ValidGraveyard Insect.YouOwn,ValidHand Insect.YouOwn,ValidLibrary Insect.YouOwn | StaticAbilities$ PerpetualToxic | Duration$ Permanent | Triggers$ Update
```

Comment:
that Remember can be write into a single "Valid..."

Reply:
UGH copied the line from Lae'zel which had the only other bad use of course – well guess both will be fixed now!